README.md

```diff
@@ -1464,6 +1465,16 @@ Type | Interface               | Value type | Description
 out  | evt.notification.report | object     | Notification reports that are sent by devices.
 out  | evt.sensor.report       | object     | Multilevel sensor reports with unknown sensor type or/and unknown unit.
 out  | evt.meter.report        | object     | Meter reports with unknown meter type or/and unit.
+in   | evt.meter.reset         | null       | Resets all historical readings.
+
+#### Service props
+
+Name               | Value example          | Description
+-------------------|------------------------|-------------
+`meter_type`       | "meter_elec", "6"      | Meter type - described as text when known and number if unknown.
```

Comment:
Probably leave one example value here and type in description that it can be either a numeric value or translated name.